### PR TITLE
Get started take 2

### PR DESCRIPTION
**Pull request by mattwelborn**
This PR reorganizes and rewords the Get Started page to emphasize the QCArchive data over the infrastructure, as discussed this morning. 

This PR replaces PR #31.

### CODE REVIEW

File: content/get_started/_index.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 date: 2019-03-11T12:35:54+06:00
 description: "Getting Started with QCArchive"
-pageDescription : "The QCArchive project has a variety of entry points
-depending on individual goals. View data in the MolSSI QCArchive database with
-our QCPortal Python API to search, analyze, and visualize results. Spin up a
-new QCFractal instance to compute your own private data or use QCEngine in your
-own ecosystem!
+pageDescription : "The QCArchive offers FAIR access to millions of computational molecular science results.
+These include curated interaction energy, thermochemical, and conformational datasets from the literature;
+data from the OpenFF initiative; and data donated to or computed by MolSSI.
```
Comment (dgasmith) [inline content/get_started/_index.md]: "data from partner groups"?

File: data/get_started.yml [hunk 2]
```diff
@@ -1,187 +1,89 @@
...
-        <a href='mailto:qcarchive@molssi.org' class='btn btn-main-md'>Contact Us</a>
-</div>
-
+compute:
+    enable: true
+    title : "Compute with QCArchive"
+    image : "images/about/openff_workflow.png"
+    content : "
+  QCArchive provides a complete platform for computing, managing, and organizing large-scale quantum chemical workflows and datasets.
+  This platform is already used by the OpenFF project, and we are actively looking
```
Comment (dgasmith) [inline data/get_started.yml]: "This platform is already used by many partner projects"

I don't fully get the "we can run your compute queue" for you from this sentence. Maybe more like:
"The platform is already by many partner groups to aggregate open data and distribute it to the community. We are actively looking to work with additional partners to generate additional open data within QCArchive for machine learning, force field prediction, methodology assessment, and more."

**Comment by mattwelborn** [comment]
Thanks for the comments! In the near future, I would like to update the Community>Partners page to include a list of academic groups who have accounts on the instance (or who we know use our data in other ways). I think this would help to demonstrate community adoption.

**Comment by dgasmith** [comment]
Sounds good, maybe separate this out as an issue?

**Pull request merged**